packages/amplication-server/package.json

```diff
@@ -105,7 +105,7 @@
     "@types/graphql-type-json": "0.3.2",
     "@types/jest": "25.2.3",
     "@types/lodash": "4.14.182",
-    "@types/node": "13.7.4",
+    "@types/node": "16.3.0",
```

Comment:
I upgrade the version because I got an error `buffer"' has no exported member 'Blob'. 23 import { Blob } from "buffer`
and I found a solution here:
https://stackoverflow.com/questions/70858113/module-buffer-has-no-exported-member-blob